app/src/main/java/tttTDD/Java/ConsoleClient.java

```diff
@@ -53,38 +50,31 @@ public Game getGame() {
     }
 
     private void setUpGame() throws Exception{
-        game = new Game(gameConfig);
+        game = new Game(gameConfig, input);
     }
 
-    private void performPlayerTurn(int value) {
-        if (game.performTurn(value)) {
-            printTheMovePerformed(value);
+    private void performPlayerTurn() {
+        if (game.performTurn(game.getCurrentPlayer())) {
+            printTheMovePerformed(game.getCurrentPlayerPreviousMove());
```

Comment:
I think I see why you called this the previous move now. Why not just call this before all the functions, that way its present tense?

pseudocode 
```
int currentMove = game.getCurrentPlayerMove(); // currently getCurrentPlayerPreviousMove
boolean result = game.performTurn(currentMove);
if success(result) {
  print(currentMove);
  print(board);
} else {
  askToPlayAgain();
}
```

Reply:
So `performTurn`'s signature was changed so that instead of receiving an input, it takes in the current player so that the player can make the move. There's also the swapping of players happening within the game rather than the console.